src/test/java/com/megazone/goop/racingcar/logic/CarTest.java

```diff
@@ -0,0 +1,32 @@
+package com.megazone.goop.racingcar.logic;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car();
+    }
+
+    @DisplayName("자동차는 랜덤값이 4 이상인 경우에 1 전진한다.")
+    @ParameterizedTest
+    @ValueSource(ints = {4, 5, 6, 7, 8, 9})
+    void moveByOneIfValueFourOrMoreTest(int randValue) {
+        assertThat(car.move(randValue)).isEqualTo(1);
```

Comment:
테스트 코드에서 `Car` 객체의 `move()`의 반환값이 무엇인지 명확하게 표현하면 더 좋을 것 같습니다.
저는 "이동한 거리"로 이해했습니다.
로컬 변수로 추출해서 변수명에 의미를 부여하면 더 명확해질 수 있습니다.
```suggestion
        int moveDistance = 1;
        assertThat(car.move(randValue)).isEqualTo(moveDistance);
```